meta/boardreports/2020-10.md

```diff
@@ -0,0 +1,68 @@
+# InnerSource Patterns WG - Report for governance meeting 2020-10
+
+**Data from June to September**
+
+## Mission - Briefly describe what your project actually does.
+
+- Discuss community InnerSource practices and dynamics, collect and document agreed upon best practices of how to do InnerSource - in the form of patterns
+- Continuously publish the most mature patterns as an ebook and website
+
+## Project/Community Status and Health - Sum up the status and health of your project and community in a few sentences
+
+- This project is seeing more activity than ~1 year ago, with 10 new contributors, some of them repeatedly, some major contributions, lots of evolutionary improvements. Also new contributions from past contributors.
+- Activity and trusted committer diversity is still too low to sustain the project in the long run on a high activity level. Low activity evolution is sustainable.
+
+## Project Activity - Describe the overall activity in the project over the past quarter.
+
+- New process proposal by @maxcapraro, discussion (@spier @mainec @newmexicokid @lenucksi) and successful adoption (initial, structured, validated, goal: faster first contribution, see contributor handbook)
+- Processing of existing content from wiki and existing PRs into publishable repo content, large scale cleanup, migration and fixing, all ongoing for the foreseeable future. 
+- Great work towards content rendering (e.g. website, initial PDFs) in progress @spier @dizquerdo (https://innersourcecommons.gitbook.io)
```

Comment:
The board can already look at the latest version if they want. Adding that link instead.

```suggestion
- Great work towards content rendering (e.g. website, initial PDFs) in progress @spier @dizquerdo (https://innersourcecommons.gitbook.io/innersource-patterns-staging/v/book/)
```